### ISSUE TITLE
Control Transparency

### ISSUE THREAD

**Issue by captkirk88** [MyraUI/Myra#30]
Please allow for controls, and their children to have a transparency property and for it to be applied.

You use SpriteBatch so it should be relatively easy.
```csharp
// in Render(RenderContext)
Batch.Draw(textureRegion, Color.White * 0.5f); // half transparency
```

**Comment by rds1983** [comment]
Thanks for the idea. It would be fixed in the next version(0.6.6)

**Comment by rds1983** [comment]
Transparency had been added in the version 0.6.6. Now Widget class has Opacity property which can take values from 0.0 to 1.0(default value is 1.0). If both container widget and its child have that property set then they are multiplied. I.e. if container opacity is 0.5 and its child opacity is 0.6. Then the child will be rendered with opacity 0.3.
